tslint-warnings.csv

```diff
@@ -149,7 +149,6 @@ CWE 710 - Coding Standards Violation"
 no-misused-new,Warns on apparent attempts to define constructors for interfaces or `new` for classes.,TSLINTL96MA6,tslint,Non-SDL,Warning,Important,Opportunity for Excellence,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,,
 no-multiline-string,Do not declare multiline strings,TSLINT10K5P9U,tslint,Non-SDL,Warning,Low,Opportunity for Excellence,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,710,"CWE 710 - Coding Standards Violation"
 no-non-null-assertion,Disallows non-null assertions using the `!` postfix operator.,TSLINTNO75FN,tslint,Non-SDL,Warning,Important,Opportunity for Excellence,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,710,"CWE 710 - Coding Standards Violation"
-no-null-undefined-union,Disallows explicitly declared or implicitly returned union types with both `null` and `undefined` as members.         ,TSLINT1JAMNL7,tslint,,,,,See description on the tslint or tslint-microsoft-contrib website,TSLint Procedure,,
```

Comment:
Not sure why this line was deleted - it is added back on my PC.
Could please run `npm install` (with npm 5+) and then `npm test` again?